### DISCUSSION TITLE
Can we do static export like in next.js ?

### DISCUSSION THREAD

**Discussion by ishank-s** [remix-run/remix#4047] [Q&A] [answered]
I wanted to integrate this in tauri. For this I would need to remove the node server

**Comment by runoncedev** [comment]
I don't think so.

**Reply by ishank-s** [reply]
Thanks!. do you think it would be a good idea? Any opinion on this?

**Reply by runoncedev** [reply]
Doesn't look like it's in Remix plan to add SSG. The whole idea of Remix is SSR on the edge.

**Comment by sergiodxa** [comment, marked as answer]
There's no built-in way, there's a [remix-ssg package](https://www.npmjs.com/package/remix-ssg) that can replace the compiler to do SSG with Remix but it's 100% experimental.

Another option could be to run your Remix app as usual, then download the HTML and other assets for all of your routes, you could use `wget` to do this on a script.
